src/bin/tikv-server.rs

```diff
@@ -650,7 +655,22 @@ fn run_raft_server(listener: TcpListener, matches: &Matches, config: &toml::Valu
     node.stop().unwrap();
 }
 
+fn reg_metrics_sig_handler() {
+    // sometimes we need to dump the metrics one by one, and compare those metrics.
+    // to flush metrics to stdout immediately, run:
+    // pkill --signal INT tikv-server
+    // or kill --s INT pid
+    Signals::set_handler(&[Signal::Int], move |_signals| {
+        let mut buffer = vec![];
+        let metric_familys = prometheus::gather();
+        let encoder = TextEncoder::new();
+        encoder.encode(&metric_familys, &mut buffer).unwrap();
+        info!("{}", String::from_utf8(buffer.clone()).unwrap());
+    });
+}
+
 fn main() {
+    reg_metrics_sig_handler();
```

Comment:
Please consider calling this function after util::run_prometheus,  for better readability.